app/src/main/res/values-pt-rPT/strings.xml

```diff
@@ -83,17 +83,17 @@
     <string name="week">Semana</string>
 
     <!-- Event types -->
-    <string name="event_types">Event types</string>
-    <string name="add_new_type">Add a new type</string>
-    <string name="edit_type">Edit type</string>
-    <string name="type_already_exists">Type with this title already exists</string>
-    <string name="color">Color</string>
-    <string name="regular_event">Regular event</string>
-    <string name="cannot_delete_default_type">Default event type cannot be deleted</string>
-    <string name="select_event_type">Select an event type</string>
+    <string name="event_types">Tipos de eventos</string>
+    <string name="add_new_type">Adicionar um tipo</string>
+    <string name="edit_type">Editar tipo</string>
+    <string name="type_already_exists">Já existe um tipo com este nome</string>
+    <string name="color">Cor</string>
+    <string name="regular_event">Evento regular</string>
+    <string name="cannot_delete_default_type">O tipo de vento padrão não pode ser apagado</string>
```

Comment:
After merge, please change this string to:

O tipo de evento padrão não pode ser apagado